app/components/Header/index.js

```diff
@@ -0,0 +1,82 @@
+/**
+ *
+ * Header
+ *
+ */
+
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import Wrapper from './Wrapper';
+import Logo from './Logo';
+import ListItem from './ListItem';
+import SearchForm from './SearchForm';
+import SearchFormInput from './SearchFormInput';
+
+import messages from './messages';
+
+/* eslint-disable react/prefer-stateless-function */
+const Header = () => {
+  const MSG_KEYS = Object.keys(messages);
+  return (
+    <Wrapper className="col-xl-8 col-lg-10">
+      <nav className="navbar navbar-expand-lg navbar-light">
+        <Link to="/" href="/">
+          <Logo />
+        </Link>
+
+        <button
+          className="navbar-toggler"
+          type="button"
+          data-toggle="collapse"
+          data-target="#navbarTogglerDemo02"
+          aria-controls="navbarTogglerDemo02"
+          aria-expanded="false"
+          aria-label="Toggle navigation"
+        >
+          <span className="navbar-toggler-icon" />
+        </button>
+
+        <div className="collapse navbar-collapse" id="navbarTogglerDemo02">
+          <ul className="navbar-nav mr-auto mt-2 mt-lg-0">
+            {MSG_KEYS.map(
+              item =>
+                messages[item].mark === 'tab' && (
+                  <ListItem className="nav-item" key={messages[item].tabCode}>
```

Comment:
<FormattedMessage /> needs to be used to render message in order for translations to work properly. I think it is OK if we just hardcode few menu items for now without iterations through the items. We will figure out later if it is needed.